src/frontend/citizen-portal/src/app/components/dispute-submit-success/dispute-submit-success.component.html

```diff
@@ -132,6 +109,16 @@ <h1>Ticket request sent successfully</h1>
         </app-alert>
       </div>
     </div>
+    <div class="row" *ngIf="isShowCheckboxes.appear_in_court.length !== noticeOfDispute.disputed_counts.length">
+      <div class="col">
+        <app-alert type="warning" icon="info">
+          <ng-container #alertContent class="alert-content">
+            <strong>You have selected that you do not want to attend a court hearing</strong>
+            You will be notified in writing to the address supplied above by the Court if your request was successful.
```

Comment:
You will be notified in writing at the address supplied above by the Court if your request was successful.

Reply:
https://justice.gov.bc.ca/jira/secure/RapidBoard.jspa?rapidView=7622&projectKey=TCVP&view=detail&selectedIssue=TCVP-1345

It is the wordings from the wireframe. Should I change that ?

Reply:
ok if it was in the wireframe like that then leave it